libs/packages/components/src/lib/external-link/external-link.directive.ts

```diff
@@ -40,6 +47,10 @@ export class ExternalLinkDirective implements OnChanges {
       }
       this.relAttr = 'noopener';
       this.targetAttr = '_blank';
+      const currentAriaLabel = this.el.nativeElement.getAttribute('aria-label');
+      if (!currentAriaLabel || currentAriaLabel.length === 0) {
+        this.ariaLabel = `Open ${this.href} in a new window`;
```

Comment:
Can you move this code outside the else block, it will not work for the internal link (FSD.gov)

Reply:
This aria label is needed for external links only - AMP throws a violation whenever there is a link with target="_blank" and does not have an aria label warning that clicking it will open a new window.

Reply:
Still, there is a possibility that they can set the target blank for the internal link as well.

Reply:
Updated to move outside the if/else statement. I used target attribute as the condition rather than isExternalLink as I didn't notice before the possibility of isExternalLink returning false but still opening on a new tab.